regress/expected/expr.out

```diff
@@ -1441,9 +1441,9 @@ RETURN ''::int
 $$) AS r(result agtype);
 ERROR:  invalid input syntax for integer: ""
 SELECT * FROM cypher('expr', $$
-RETURN 'falze'::int
+RETURN 'false_'::int
```

Comment:
I'm hoping this is an acceptable change. While `falze` != `false`, `false_` != `false` too, and it has the benefit of not being flagged by spell checking tools.

I'm happy to drop this change or any other change.